README.md

```diff
@@ -36,6 +36,10 @@ _Note: Simplenote API features such as sharing and publishing will not work with
 
 ## Contributing
 
-Follow the same guidelines as [WordPress for iOS](https://make.wordpress.org/mobile/handbook/pathways/ios/how-to-contribute/).
+Read our [Contributing Guide](CONTRIBUTING.md) to learn about reporting issues, contributing code, and more ways to contribute.
```

Comment:
I looks like the `CONTRIBUTING.md` file actually has a trailing space in the filename causing this reference to show a 404.  Try the links below for verification.

Pass: https://github.com/Automattic/simplenote-ios/blob/275ebf7763eae8e385e363b1b0b756350790e884/CONTRIBUTING.md%20
Fail: https://github.com/Automattic/simplenote-ios/blob/275ebf7763eae8e385e363b1b0b756350790e884/CONTRIBUTING.md

We should be able to fix that by just removing the trailing space from the `CONTRIBUTING.md` filename.

Reply:
Nice catch @theck13 → the file has been renamed. 🙇